release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -497,6 +497,12 @@ For more information, see xref:../networking/ptp/about-ptp.adoc#ptp-dual-ports-o
 
 With this update, you can specify larger kernel page sizes to improve performance for memory-intensive, high-performance workloads on ARM infrastructure nodes with the realtime kernel disabled. For more information, see xref:../scalability_and_performance/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-configuring-kernal-page-size_cnf-low-latency-perf-profile[Configuring kernel page sizes].
 
+[id="ocp-release-notes-tuning-hcp-performance-profile_{context}"]
+==== Tuning {hcp} using a performance profile
+
+With this release, you can now tune {hcp} for low latency by applying a performance profile. For more information, see xref:../scalability_and_performance/cnf-tuning-low-latency-hosted-cp-nodes-with-perf-profile.adoc#cnf-create-performance-profiles-hosted-cp[Creating a performance profile for hosted control planes].
```

Comment:
Minor comment; the previous note starts with "With this update", while this one begins "With this release". I would probably make the language consistent here.

Reply:
I might also update the language "for low latency" to something like "to achieve low latency" since it's not immediately clear I think what "for" means here.